documentation/openapi-authoring-automated-guidelines.md

```diff
@@ -2239,15 +2239,15 @@ Eg:
 
 Links: [Index](#index) | [Error vs. Warning](#error-vs-warning) | [Automated Rules](#automated-rules) | [ARM](#arm-violations): [Errors](#arm-errors) or [Warnings](#arm-warnings) | [SDK](#sdk-violations): [Errors](#sdk-errors) or [Warnings](#sdk-warnings)
 
-### <a name="r4005" ></a>R4005 XmsEnumNameUnique
+### <a name="r4005" ></a>R4005 UniqueXmsEnumName
 
-**Category** : ARM Warning
+**Category** : ARM Error
 
-**Applies to** : ARM OpenAPI(swagger) specs
+**Applies to** : ARM and Data plane OpenAPI(swagger) specs
```

Comment:
let's apply to ARM only by default.  Apply to Data plane only on request.  @akning-ms what do you think?

Reply:
offline synced